liquibase-standard/src/main/java/liquibase/changelog/DatabaseChangeLog.java

```diff
@@ -1106,7 +1108,7 @@ public static String normalizePath(String filePath) {
         we fall back to path normalization using Paths.get(), which might fail on Windows.
          */
         if (normalized == null) {
-            normalized = Paths.get(filePath).normalize().toString();
+            normalized = normalizePathViaPaths(filePath, true);
```

Comment:
Hey @andrus,

Sonar scan is reporting an issue here, because  `normalizePathViaPaths()` can return a null value and it could ends up in NPE with the following code execution. I think we would need to add a `NULL` check to avoid that happening.

Reply:
Not sure if you are able to see issues reported by Sonar in the following [link](https://sonarcloud.io/project/issues?issueStatuses=OPEN%2CCONFIRMED&pullRequest=5894&id=liquibase).

Thanks,
Daniel.

Reply:
It only returns null when filePath is null, and there's already a check for that at the top of `normalizePath`. Any non-null path will result in a non-null return. So while Sonar is formally correct, it is being too paranoid :) 

None the less, I just pushed a change with a null check